### PR TITLE
More dream sources

### LINKED ISSUE #29: Fetches don't count for all color sources
I was tuning [this list](https://mtgoncurve.com/?v0=eyJjb2RlIjoiMSBBbmFmZW56YSwgS2luLVRyZWUgU3Bpcml0XG4xIEFyaWQgTWVzYVxuMSBCYW5pc2hlciBQcmllc3RcbjEgQmF0dGxlZmllbGQgRm9yZ2VcbjEgQmVuZXZvbGVudCBCb2R5Z3VhcmRcbjEgQmxpbmttb3RoIE5leHVzXG4xIEJsb29kc3RhaW5lZCBNaXJlXG4xIEJvbWF0IENvdXJpZXJcbjEgQnJ1c2UgVGFybCwgQm9vcmlzaCBIZXJkZXJcbjEgQ2FzdGxlIEFyZGVudmFsZVxuMSBDaGFuZHJhLCBBY29seXRlIG9mIEZsYW1lXG4xIENoYXJtaW5nIFByaW5jZVxuMSBDaXR5IG9mIEJyYXNzXG4xIENpdHkgb2YgVHJhaXRvcnNcbjEgQ2xpZmZ0b3AgUmV0cmVhdFxuMSBDb21tYW5kIFRvd2VyXG4xIENyeXN0YWwgVmVpblxuMSBEcmFubml0aCBNYWdpc3RyYXRlXG4xIEVhcnRoc2hha2VyIEtoZW5yYVxuMSBFaWdhbmpvIENhc3RsZVxuMSBFdGhlcnN3b3JuIENhbm9uaXN0XG4xIEZpZW5kIEh1bnRlclxuMSBGaWd1cmUgb2YgRGVzdGlueVxuMSBGbG9vZGVkIFN0cmFuZFxuMSBGb3JjZSBvZiBWaXJ0dWVcbjEgR2l2ZXIgb2YgUnVuZXNcbjEgR29ibGluIEJvbWJhcmRtZW50XG4xIEdvYmxpbiBCdXNod2hhY2tlclxuMSBHb2JsaW4gQ3JhdGVybWFrZXJcbjEgR29ibGluIEd1aWRlXG4xIEdvYmxpbiBJbnN0aWdhdG9yXG4xIEdvYmxpbiBSYWJibGVtYXN0ZXJcbjEgR3JhbmQgQWJvbGlzaGVyXG4xIEdyaW0gSW5pdGlhdGVcbjEgSWNlaGlkZSBHb2xlbVxuMSBJbnNwaXJpbmcgVmFudGFnZVxuMSBJc2FtYXJ1LCBIb3VuZCBvZiBLb25kYVxuMSBKZXdlbGVkIEFtdWxldFxuMSBLYXJnYW4gSW50aW1pZGF0b3JcbjEgTGVnaW9uIExveWFsaXN0XG4xIExlZ2lvbiBXYXJib3NzXG4xIExlZ2lvbidzIExhbmRpbmdcbjEgTGlnaHRuaW5nIEJvbHRcbjEgTG90dXMgUGV0YWxcbjEgTG95YWwgQXBwcmVudGljZVxuMSBMdW1pbm91cyBCcm9vZG1vdGhcbjEgTWFndXMgb2YgdGhlIE1vb25cbjEgTWFuYSBDb25mbHVlbmNlXG4xIE1hcnNoIEZsYXRzXG4xIE1vdGhlciBvZiBSdW5lc1xuMSBOZWVkbGV2ZXJnZSBQYXRod2F5IFxuMSBQYWxhY2UgSmFpbGVyXG4xIFBoeXJleGlhbiBSZXZva2VyXG4xIFBpYSBhbmQgS2lyYW4gTmFsYWFyXG4xIFBpYSBOYWxhYXJcbjEgUGxhdGVhdVxuMSBQcmlzbWF0aWMgVmlzdGFcbjEgUmFpc2UgdGhlIEFsYXJtXG4xIFJhbmdlci1DYXB0YWluIG9mIEVvc1xuMSBSZW1vcnNlZnVsIENsZXJpY1xuMSBSZXZlcmVudCBNYW50cmFcbjEgQmFzcmkncyBMaWV1dGVuYW50XG4xIFJ1Z2dlZCBQcmFpcmllXG4xIFNhY3JlZCBGb3VuZHJ5XG4xIFNhbXVyYWkgb2YgdGhlIFBhbGUgQ3VydGFpblxuMSBTY2FsZGluZyBUYXJuXG4xIFNlYXNvbmVkIFB5cm9tYW5jZXJcbjEgU2VsZmxlc3MgU2F2aW9yXG4xIFNlbGZsZXNzIFNwaXJpdFxuMSBTaGF0dGVyc2t1bGwsIHRoZSBIYW1tZXIgUGFzc1xuMSBTaW1pYW4gU3Bpcml0IEd1aWRlXG4xIFNrdWxsY2xhbXBcbjEgU2t5Y2xhdmUgQXBwYXJpdGlvblxuMSBTa3ljbGF2ZSBDbGVyaWMgXG42IFNub3ctQ292ZXJlZCBNb3VudGFpblxuOSBTbm93LUNvdmVyZWQgUGxhaW5zXG4xIFN0b25lY29pbCBTZXJwZW50ICMgWD0zXG4xIFN0b25lZm9yZ2UgTXlzdGljXG4xIFN1bmJha2VkIENhbnlvblxuMSBUaGFsaWEsIEd1YXJkaWFuIG9mIFRocmFiZW5cbjEgVGhhbGlhLCBIZXJldGljIENhdGhhclxuMSBUaGFsaWEncyBMaWV1dGVuYW50XG4xIFRpdGhlIFRha2VyXG4xIFdpbmRzd2VwdCBIZWF0aFxuMSBXaW5vdGEsIEpvaW5lciBvZiBGb3JjZXNcbjEgV29vZGVkIEZvb3RoaWxsc1xuMSBadXJnbyBCZWxsc3RyaWtlciIsIm9uX3RoZV9wbGF5IjpmYWxzZSwiaW5pdGlhbF9oYW5kX3NpemUiOjcsIm11bGxpZ2FuX2Rvd25fdG8iOjUsIm11bGxpZ2FuX29uX2xhbmRzIjpbMCwxLDYsN10sImNhcmRzX3RvX2tlZXAiOiIifQ%3D%3D#) and I saw that the fetches didn't seem to count toward both colors I used (red and white).

image

I doesn't seem to need much change to take it into action. I saw that you use Scryfall API and it appears to have a `is:fetchland` possible query. It might bring much help to everyone using your wonderful app

Cheers

### PR DESCRIPTION
**Pull request by Spigushe**
Updated existing map :
- Changed fectchlands to dream lands
- Added some more fectchlands : Prismatic Vista, Terramophic Expanse
- Added Commander Lands as dreams lands : Command Tower, Opal Palace, Path of Ancestry

Closes #29

### CODE REVIEW

**Comment by Spigushe** [comment]
This PR adresses the issue referenced at #29

File: lib/src/scryfall.rs [hunk 1]
```diff
@@ -267,46 +267,58 @@ lazy_static! {
...
+        ),
+        (
+            "Path of Ancestry",
+            ManaCost::from_rgbuwc(1, 1, 1, 1, 1, 0)
+        ),
         // fetch lands
         (
             "Arid Mesa",
-            ManaCost::from_rgbuwc(1, 0, 0, 0, 1, 0)
+            ManaCost::from_rgbuwc(1, 1, 1, 1, 1, 0)
```
Comment (jshrake) [inline lib/src/scryfall.rs:285]: I'm not sure I understand why the Zendikar fetch lands should count for more than their two mana color pips. For example, https://gatherer.wizards.com/pages/card/details.aspx?multiverseid=177584 should only count as red and white?

Comment (Spigushe) [reply lib/src/scryfall.rs:285]: You can find every dual land, shockland, snow dual either mountain or plains and so on for every fetchland.

**Comment by jshrake** [comment]
Thanks for your contribution and patience! I had to resolve the merge conflict and merge locally. https://github.com/mtgoncurve/landlord/commit/e310fea001302f670b5360ff74c3eeba78354762. I'll get this deployed to mtgoncurve.com asap.

**Pull request closed without merging**